encoding/layout/prefix.rs

```diff
@@ -82,42 +82,42 @@ impl Prefix {
 }
 
 make_prefix_enum! {
-    // Reserved: 0-9
-    VertexEntityType => 10, true;
-    VertexRelationType => 11, true;
-    VertexAttributeType => 12, true;
-    VertexRoleType => 15, true;
-    DefinitionStruct => 20, true;
-    DefinitionFunction => 21, true;
+    // Reserved: 0x00-0x09
+    VertexEntityType => 0x0A, true;
+    VertexRelationType => 0x0B, true;
+    VertexAttributeType => 0x0C, true;
+    VertexRoleType => 0x0F, true;
+    DefinitionStruct => 0x14, true;
+    DefinitionFunction => 0x15, true;
```

Comment:
I usually only look here when reading a raw dump of a key or a commit, which is in hex.

The values were replaced automatically with `printf("0x%02X", num)`, with no changes.